release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -506,6 +506,16 @@ For more information, see xref:../scalability_and_performance/cnf-numa-aware-sch
 
 For more information about the feature, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-using-pgt-to-configure-power-saving-states_ztp-advanced-policy-config[Configuring power states using PolicyGenTemplates CRs].
 
+[id="ocp-4-13-scalability-and-performance-talm-updates"]
+==== Pre-caching container images for updates
+
+For container image pre-caching, {cgu-operator-first} compares the available disk space with the estimated {product-title} image size to ensure that there is enough space. 
+If a cluster has insufficient space, {cgu-operator} cancels pre-caching for that cluster and does not remediate policies on it.
+
+The pre-caching feature typically downloads more images than a cluster needs for an update. You can use the `excludePrecachePatterns` field in the `configmap` to control which pre-cache images {cgu-operator} downloads to a cluster.
```

Comment:
These lines can be removed if you accept the suggested rewrite.

Reply:
done